src/app.js

```diff
@@ -1 +1,53 @@
-'use strict';
+import readline from 'readline';
```

Comment:
You don't need any extra modules to solve this task

Reply:
I used readline in order not to disable linter rules for console, and as I know github tests are not necessary